src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -39,7 +40,7 @@ using namespace OpenRCT2;
 
 static constexpr const StringId WINDOW_TITLE = STR_NONE;
 constexpr int32_t WINDOW_SCENERY_MIN_WIDTH = 634;
-constexpr int32_t WINDOW_SCENERY_MIN_HEIGHT = 180;
+constexpr int32_t WINDOW_SCENERY_MIN_HEIGHT = 195;
```

Comment:
Why does the window need to be taller?

Reply:
Moving the buttons on the side down means we need to increase the minimum height to compensate

Reply:
To clarify, this is how it looks in current develop:
![image](https://user-images.githubusercontent.com/3720978/215812829-c41b21b9-ade4-42d7-beb3-adb1a9c0d22d.png)

And here's after this PR:
![image](https://user-images.githubusercontent.com/3720978/215813320-eb12b5ad-f7c8-4dd6-a34f-8f257f531a0f.png)

If you want to keep the side buttons in the old position this can be reverted but I thought it looked nicer having them aligned with the scroller.